### PR TITLE
Fix "imported but not used" linters

### PR DESCRIPTION
**Pull request by Blast545**
These linter errors started showing in the Galactic coverage jobs three days ago, reference job:
https://ci.ros2.org/job/nightly_linux_galactic_coverage/585/

As it was a quick one, I decided to open a PR to fix it. PTAL @Crola1702 @cottsay 

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

**Review by mjcarroll — APPROVED** [review]

**Comment by Blast545** [comment]
I think it might be worth taking a look as to why this started appearing in the coverage jobs, and hasn't shown in the release/debug jobs. @Crola1702

**Comment by Blast545** [comment]
Galactic up to `launch_ros`, Linux CI:

[Build Status](https://ci.ros2.org/job/ci_linux/17721/)

**Review by cottsay — CHANGES_REQUESTED** [review]
Both of these are caused by typing annotations, and will appear with older Python interpreters. They should be marked with `#noqa`, not removed.

Here's where they're used:
https://github.com/ros2/launch_ros/blob/3c355ed49ba5e746d5e5ca454390bf6b6ac2fc2d/launch_ros/launch_ros/actions/load_composable_nodes.py#L82
https://github.com/ros2/launch_ros/blob/3c355ed49ba5e746d5e5ca454390bf6b6ac2fc2d/launch_ros/launch_ros/utilities/evaluate_parameters.py#L155

**Comment by Crola1702** [comment]
Running CI with new change: [Build Status](https://ci.ros2.org/job/ci_linux/17726/)

File: launch_ros/launch_ros/actions/load_composable_nodes.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 from typing import List
 from typing import Optional
-from typing import Text
+from typing import Text # noqa
```
Comment (cottsay) [inline launch_ros/launch_ros/actions/load_composable_nodes.py:21]:
```suggestion
from typing import Text  # noqa: F401
```

File: launch_ros/launch_ros/utilities/evaluate_parameters.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 from collections.abc import Mapping
 from collections.abc import Sequence
-import pathlib
+import pathlib # noqa
```
Comment (cottsay) [inline launch_ros/launch_ros/utilities/evaluate_parameters.py:20]:
```suggestion
import pathlib  # noqa: F401
```

**Review by cottsay — CHANGES_REQUESTED** [review]
We can disable the specific check that's failing. Additionally, flake8 calls for two spaces before an inline comment, but the unqualified `#noqa` was actually squelching that violation as well.

**Comment by Crola1702** [comment]
Running CI again with the latest commits:
[Build Status](https://ci.ros2.org/job/ci_linux/17759/)

**Review by cottsay — APPROVED** [review]

**Pull request merged**